pkg/translator/signalfx/from_metrics.go

```diff
@@ -199,6 +215,57 @@ func convertHistogram(in pmetric.HistogramDataPointSlice, name string, mt *sfxpb
 	return dps.out
 }
 
+func convertExponentialHistogram(in pmetric.ExponentialHistogramDataPointSlice, name string, mt *sfxpb.MetricType, extraDims []*sfxpb.Dimension) []*sfxpb.DataPoint {
+	var numDPs int
+	for i := 0; i < in.Len(); i++ {
+		histDP := in.At(i)
+		if histDP.HasSum() {
+			numDPs++
+		}
+
+		if histDP.HasMin() {
+			numDPs++
+		}
+
+		if histDP.HasMax() {
+			numDPs++
+		}
+	}
+	dps := newDpsBuilder(numDPs)
+
+	for i := 0; i < in.Len(); i++ {
+		histDP := in.At(i)
+		ts := fromTimestamp(histDP.Timestamp())
+		dims := attributesToDimensions(histDP.Attributes(), extraDims)
+
+		countDP := dps.appendPoint(name+"_count", mt, ts, dims)
+		count := int64(histDP.Count())
+		countDP.Value.IntValue = &count
+
+		if histDP.HasSum() {
+			sumDP := dps.appendPoint(name+"_sum", mt, ts, dims)
+			sum := histDP.Sum()
+			sumDP.Value.DoubleValue = &sum
+		}
+
+		if histDP.HasMin() {
+			// Min is always a gauge.
+			minDP := dps.appendPoint(name+"_min", &sfxMetricTypeGauge, ts, dims)
+			min := histDP.Min()
+			minDP.Value.DoubleValue = &min
+		}
+
+		if histDP.HasMax() {
+			// Max is always a gauge.
+			maxDP := dps.appendPoint(name+"_max", &sfxMetricTypeGauge, ts, dims)
+			max := histDP.Max()
+			maxDP.Value.DoubleValue = &max
+		}
+	}
+
+	return dps.out
+}
```

Comment:
I'm not sure we should provide Exponential histogram support with the limited functionality (no buckets and few other fields) and no test coverage. I'd say let's better show an error and drop data points until there is a real ask for this or OTLP support on the backend

Reply:
Thanks, I will address this asap.